xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/dashboard/dashboard.css

```diff
@@ -102,3 +102,9 @@
   min-height: 16px;
   padding: 3px 6px 3px 22px;
 }
+
+/* Hide gadget title, when wider than gadget width */
+.gadget-title.wikigeneratedheader {
```

Comment:
My bad. Thanks for noticing! I corrected it [here](https://github.com/gabrielasmeria/xwiki-platform/commit/e4204bfd371324fab83f63172b040a3710329e00).